src/app/patient/handlers/request/dto.go

```diff
@@ -0,0 +1,32 @@
+package request
+
+import (
+	"clinic-api/src/app/patient"
+	"clinic-api/src/types"
+	"clinic-api/src/utils"
+
+	"github.com/google/uuid"
+)
+
+type Request struct {
+	Name      string `json:"name"`
+	NIK       string `json:"nik"`
+	Phone     string `json:"phone"`
+	Address   string `json:"address"`
+	DOB       string `json:"dob"`
+	Gender    string `json:"gender"`
+	BloodType string `json:"blood_type"`
+}
+
+func (req *Request) MapToDomain() patient.Domain {
+	return patient.Domain{
+		ID:        uuid.Nil,

```

Comment:
Prefer not to specify this, cause it will not be used in `record.go`

Reply:
Implemented at https://github.com/ALTERA-CAPSTON-41/backend/pull/38/commits/8e2b69069a1b40f3059ff9cc80045a2727151867.